sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -65,9 +66,10 @@ export const studentRoutes: Array<RouteRecordRaw> = [
         component: StudentProfile,
       },
       {
-        path: "dynamic-application-form",
+        path: "dynamic-application-form/:id?",
```

Comment:
This is going to be visible in the browser as "dynamic-applcation-form/:id" right? Can we name it just "application form". User shouldn't care/know about the "dynamic"

Reply:
The ":id" is not going to be visible in the browser.
The URL is going to be something like "dynamic-applcation-form/1" or "dynamic-applcation-form/2".
There is no requirement for it right now and this was introduced to us to be able to test and demo that we can save and retrieve the dynamic data from DB.

Reply:
The point here was about the word "dynamic"

Reply:
Oh, I got it now, I can rename it. I am not sure if It should replace the current one on Vue or create a new one. I will adjust the name and keep the current one alive for now.

Reply:
Route renamed to "application-form" for now as requested.